frontend/util.lua

```diff
@@ -563,6 +563,61 @@ function util.makePath(path)
     return lfs.mkdir(path)
 end
 
+--- As `rm`
+-- @string path of the file to remove
+-- @treturn bool true on success; nil, err_message on error
+function util.removeFile(file)
+    local lfs = require("libs/libkoreader-lfs")
+    if file and lfs.attributes(file, "mode") == "file" then
+        return os.remove(file)
+    elseif file then
+        return nil, file .. " is not a file"
+    else
+        return nil, "file is nil"
+    end
+end
+
+-- Gets total, used and available bytes for a given directory.
+-- If the directory does not exists (or any other error happens) it
+-- will return fallback values.
```

Comment:
I'm not sure I quite understand the fallback here. If it needs to be a table with numbers and can't be something like false or nil, how about `-1` for example?

Reply:
Ok. We need the fallback values in calibre but it can return nil here and in calibre use the fallback.

Reply:
Sounds good!